Core/composer.json

```
@@ -12,7 +12,7 @@
         "guzzlehttp/psr7": "^2.6",
         "monolog/monolog": "^2.9|^3.0",
         "psr/http-message": "^1.0|^2.0",
-        "google/gax": "^1.27.0"
+        "google/gax": "^1.28.2"
```

Update this to use the `main` branch of `gax-php` just to run the tests before we release it.
```suggestion
        "google/gax": "dev-main as 1.29"
```